libraries/tools/kotlin-stdlib-gen/src/templates/Elements.kt

```diff
@@ -14,7 +14,7 @@ fun elements(): List<GenericFunction> {
         doc { f -> "Returns `true` if [element] is found in the ${f.collection}." }
         typeParam("@kotlin.internal.OnlyInputTypes T")
         returns("Boolean")
-        body(Iterables) { f ->
+        body(Iterables) {
```

Comment:
Please extract unrelated changes, such as formatting improvement in a separate PR.